### ISSUE TITLE
datalad crawl-init results in runtime Error - Loris template

### ISSUE THREAD

**Issue by mathdugre** [datalad/datalad-crawler#59]
### Summary
I was trying to follow the [example from the doc](https://datalad-crawler.readthedocs.io/en/latest/demos/track_data_from_webpage.html) to initialize a datalad crawler, however, I am running into a RuntimeError.

### Context:
I installed datalad-crawler from [this branch](https://github.com/driusan/datalad-crawler/tree/enh-loriscrawler) since I need the LORIS crawler.

I executed this command:
**note that the URL I use requires authentification and is on a private network**
```bash
datalad crawl-init --template=loris url=some.url.ca
```

Error I get:
```bash
[ERROR  ] Failed to import pipeline from /Users/math/Documents/opensource/RA-perform/test/datalad-crawler/datalad_crawler/pipelines/loris.py: 
cannot import name 'UnhandledRedirectError' from 'datalad.downloaders.base' (/Users/math/Documents/opensource/RA-perform/venv/lib/python3.8/site-packages/datalad/downloaders/base.py)
[crawl_url.py:<module>:17] [pipeline.py:load_pipeline_from_module:403] (RuntimeError) 
```

**Comment by kyleam** [comment]
That failure is caused by an upstream change in datalad and is fixed in master as of ebdcd73 (MNT: Adjust imports for compatibility with developmental DataLad, 2019-08-27).

To resolve the failure you could bring master's changes into your local loris branch by either rebasing the loris branch on top of master or merging master into the loris branch.  (I just did a rebase locally to check if there are any conflicts you'd need to resolve, and there weren't.)

**Comment by mathdugre** [comment]
Thank you, this worked as mentioned.
